lib/app/modules/settings/views/weather_api.dart

```diff
@@ -22,6 +22,7 @@ class WeatherApi extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    var _validate = false.obs;
```

Comment:
Please move this to the controller and access it from there, we try to keep the UI and the business logic separate.

Reply:
Also, great job! I assume you're new to GetX as well, so definitely well done!

Just remember to resolve merge conflicts as well!